docker-compose.yaml

```diff
@@ -2,35 +2,26 @@ version: "3.8"
 services:
   d4:
     image: dillonad/d4bot:latest
+    build:
+      context: .
+      dockerfile: Dockerfile
     container_name: d4
-    env_file:
-      - .env
-      - .env.local
-    networks:
-      - d4bot
-
-  otel-collector:
-    container_name: otel-collector
-    image: ghcr.io/open-telemetry/opentelemetry-collector-releases/opentelemetry-collector-contrib:latest
-    volumes:
-      - ./otel-config.yaml:/etc/otelcol-contrib/config.yaml:Z
-    ports:
-      - "4317:4317"
```

Comment:
I think we need the OTEL collector to translate between the OpenTelemetry format and the Jaeger format.

Reply:
They don't advertise it much, but the all-in-one executable will consume http / grpc OTLP natively. https://www.jaegertracing.io/docs/1.46/getting-started/#all-in-one

Reply:
\* angrily shakes fist at jaeger *